great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -38,6 +68,52 @@ class AbstractDataContext(ABC):
         "/etc/great_expectations.conf",
     ]
     DOLLAR_SIGN_ESCAPE_STRING = r"\$"
+    TEST_YAML_CONFIG_SUPPORTED_STORE_TYPES = [
```

Comment:
At some point to clean up the length of the data context classes I would love to see these lists and `test_yaml_config` itself moved to a separate class which is invoked from the data context.

Reply:
I really like this idea.  I started putting the following note throughout the code. I'll finish up in the morning

  `# NOTE: <DataContextRefactor>`